extensions/axon/deployment/src/test/README.md

```diff
@@ -0,0 +1,10 @@
+The unit tests for this extension are ignored because the Axon framework can't be run in embedded mode.
+
+To run this tests its required to start an Axon server instance on localhost.
+
+You can start that with Docker:
+```
+docker run -d --name my-axon-server -p 8024:8024 -p 8124:8124 axoniq/axonserver
```

Comment:
I think we need all this to move to an integration test that will be like the rest of the Quarkus integration tests. 
Plenty of the integration tests already use docker to launch a dependency - see for example: https://github.com/geoand/quarkus/blob/c9cba824e8812fa3f15474b8382ac5d90f7238aa/integration-tests/jpa-postgresql/pom.xml#L158